packages/main/src/ToggleButton.ts

```diff
@@ -44,7 +44,14 @@ class ToggleButton extends Button {
 	@property({ type: Boolean })
 	pressed!: boolean;
 
-	_onclick() {
+	_onclick(e: MouseEvent) {
+		if (this.disabled) {
+			e.preventDefault();
+			e.stopPropagation();
+
+			return;
+		}
+
```

Comment:
Also I saw this conversation for an older [fix](https://github.com/SAP/ui5-webcomponents/pull/6400/files/967351b5c35b0afd0d666d41063de5247f404646#r1105481069) There it's mentioned that the click should be stopped, but the hover - not (which sounds strange to me), but Todor could recall what happened in the fix.

<img width="1149" alt="Screenshot 2024-07-15 at 12 20 50" src="https://github.com/user-attachments/assets/999db301-2d6d-4bc6-9494-e0bbd65c9412">

just to make sure that while fixing this we don't reopen/open another bug.